server/utils/users/data/rolesToPermissions.json

```diff
@@ -0,0 +1,186 @@
+[
+  {
+    "name": "assessor",
+    "permissions": [
+      "cas1_assess_appealed_application",
+      "cas1_assess_application",
+      "cas1_assess_placement_application",
+      "cas1_view_assigned_assessments"
+    ]
+  },
+  {
+    "name": "matcher",
+    "permissions": [
+      "cas1_assess_placement_application"
+    ]
+  },
+  {
+    "name": "future_manager",
```

Comment:
Maybe add a comment that this will be renamed in the future once legacy roles have been removed?

Reply:
This is covered in the Confluence page about roles and permissions -- besides, this file is copied over from the API, and also comments don't work well in JSON 😄